app/views/publisher_information/publisher_updates.govspeak.md

```diff
@@ -1,3 +1,20 @@
+
+### Email links, content data, and guidance updates <span class="govuk-caption-m">16 May 2019</span>
+Added a link to [Content Data](https://content-data.publishing.service.gov.uk/content) from the edit screen so you can easily check the performance of published content.
+
+You can now make email links with the link button. Select an email address and the link button will add email markdown.
+
+Updated styling of markdown preview to help you see the difference between editing and preview more easily.
+
+Added [guidance to help users familiar with Whitehall publisher get started](/how-to-use-publisher).
+
+We have also updated the guidance on 
```

Comment:
You need a line break before these

![Screen Shot 2019-05-15 at 16 31 20](https://user-images.githubusercontent.com/282717/57788356-e5da4a00-772e-11e9-8d1f-aca73f279c4e.png)

Reply:
No YOU need a line break